fixtures/flight/src/Form.js

```diff
@@ -1,25 +1,26 @@
 'use client';
 
 import * as React from 'react';
+import ErrorBoundary from './ErrorBoundary.js';
 
 export default function Form({action, children}) {
   const [isPending, setIsPending] = React.useState(false);
 
   return (
-    <form
-      action={async formData => {
-        setIsPending(true);
-        try {
-          const result = await action(formData);
-          alert(result);
-        } catch (error) {
-          console.error(error);
-        } finally {
-          setIsPending(false);
-        }
-      }}>
-      <input name="name" />
-      <button>Say Hi</button>
-    </form>
+    <ErrorBoundary>
+      <form
+        action={async formData => {
+          setIsPending(true);
```

Comment:
Does this even work now given that this whole thing should be a transition now?

Reply:
Ah yeah I guess it doesn't. I could wrap it in flushSync temporarily. Or maybe just wait to update the fixture until I add useFormPending.

Reply:
OTOH that behavior is broken anyway. So I'l do the flushSync for now, with a TODO explaining it's temporary.

Reply:
The `isPending` state is dead code anyways. I think it's copy & paste from the button example where it controlled the disabled state.

Reply:
I think it's just there to illustrate what the pattern is. Everything resolves so fast anyway you don't notice in the button example either.

Reply:
Yeah, when I introduced it in https://github.com/facebook/react/pull/26293/files there was a delay of 500ms in the action. 🤷